### DISCUSSION TITLE
how to get OSQP_VERSION by programming?

### DISCUSSION THREAD

**Discussion by james1kobby** [osqp/osqp#628] [C Interface] [answered]
I found # define OSQP_VERSION ("0.6.2") in constant.h, and I wanted to ask if there is any interface in OSQP that can be used to get this version number, how to program it, and if you can give an example

**Comment by goulart-paul** [comment, marked as answer]
There is an `osqp_version()` function.   

https://github.com/osqp/osqp/blob/02a117cfc8ad21b06c2596603a2046ee61c82786/src/osqp_api.c#L54

**Reply by imciner2** [reply]
Note that this function is new in the 1.0 version, I believe. The 0.6 releases only have the define statement giving a version string.
